unfoldingWord/translationCore
# Feature/2528/upgrade language selection

#### This pull request addresses:

Issue: #2528
Also fixes: https://github.com/unfoldingWord-dev/translationCore/issues/3240


#### How to test this pull request:

Go into Projects, and for one of the project cards, click on menu and select  'Edit Project Details'.
For both Language Name and Language Code: when you start typing letters you should see a drop down menu with partial matches.  If the Language name or Language code does not match an entry in langnames.json, you should see an error.  Once you select a valid value (by either typing an entry and hitting enter, or by selecting from menu), all the language entries will be updated.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/3394)
